test/test_dense/test_sketch_gefl3.cc

```diff
@@ -210,8 +210,195 @@ class TestLSKGE3 : public ::testing::Test
         );
     }
 
+    template<typename T>
+    static void test_sketch_vec_wide(
+        uint32_t seed, // Seed for S_wide
+        int64_t d,     // Dim of y_wide and y_ref
+        int64_t m,     // Dim of x. Expect m > d.
+        int64_t incx,  // Stride between elements of x
+        int64_t incy   // Stride between elements of y_wide and y_ref.
+    ) {
+        T total = 0;  
+        T x[incx*m];
+        T y_wide[incy*d];
+        T y_ref[incy*d];
+        
+        // Initialize elemnts of x to 1 with stride of incx between elements
+        for (int i = 0; i < incx*m; i++){
+            x[i] = 1.0;
+        }
+
+        // Generate wide sketching operator
+        RandBLAS::DenseDist D(d, m);
+        RandBLAS::DenseSkOp<double> S_wide(D, seed, nullptr);
```

Comment:
If you template on ``T`` then you should use ``RandBLAS::DenseSkOp<T>``, not ``RandBLAS::DenseSkOp<double>``.